src/envoy/http/jwt_auth/auth_store.h

```diff
@@ -86,6 +87,8 @@ class JwtAuthStoreFactory : public Logger::Loggable<Logger::Id::config> {
  private:
   // The auth config.
   JwtAuthenticationConstSharedPtr config_;
+  // A dummy Auth store to verify config is valid
+  JwtAuthStore dummy_store_;
```

Comment:
I see dummy_store_ constructed but never used.   What's the purpose?

Reply:
I believe this is used to validate the config during its construction: see https://github.com/istio/proxy/pull/2298#discussion_r299562081
@qiwzhang